src/core_plugins/testbed/public/testbed.html

```diff
@@ -4,7 +4,37 @@
     <!-- All markup within the CONTENT section below can be deleted and replaced with whatever markup
     you need to demonstate your functionality. Nothing need be preserved. -->
     <!-- CONTENT START -->
+    <table class="kuiTable kuiVerticalRhythm">
+      <thead>
+        <tr class="kuiTableRow">
+          <th class="kuiTableHeaderCell kuiTableHeaderCell--checkBox"></th>
+          <th class="kuiTableHeaderCell">Item</th>
+        </tr>
+      </thead>
+      <tr
+        ng-repeat="item in testbed.myItems"
+        class="kuiTableRow"
+      >
+        <td class="kuiTableRowCell kuiTableRowCell--checkBox">
+          <div class="kuiTableRowCell__liner">
+            <selectable-row
+              id="item.id"
+              is-selected="item.isSelected"
+              on-select-change="testbed.onSelectChange"
+            ></selectable-row>
```

Comment:
Looks good! Thought tbh, from looking at the interface and implementation, wouldn't it make more sense to just call this directive `check-box` or `kui-check-box`? I think we'll end up implementing a React component that does the same thing, and we'll probably just call it `KuiCheckBox` because that's pretty much all it's concerned with. It could be used in a table row, a form, etc.